governance/program/src/state/proposal.rs

```diff
@@ -271,13 +271,19 @@ impl Proposal {
     }
 }
 
-/// Returns current vote threshold in relation to the total governing token supply
-fn get_vote_threshold(vote_count: u64, governing_token_supply: u64) -> u64 {
-    (vote_count as u128)
-        .checked_mul(100)
-        .unwrap()
-        .checked_div(governing_token_supply as u128)
-        .unwrap() as u64
+/// Returns current vote threshold percentage in relation to the total governing token supply
+fn get_vote_threshold_percentage(vote_count: u64, governing_token_supply: u64) -> f64 {
```

Comment:
I defined the threshold as the level the vote must go over and not just to be equal in order to`Succeed`. In other words the vote must be `yes_threshold`% + 1 vote to pass. 
Let say if you defined  `yes_threshold` as 50% then  it means you need 50% + 1 Yes votes to pass because that gives you majority.
Exact and even 50/50 split (or any other like 60/40)  to me is undecided and the extra +1 vote is required to break the tie. 
If on the other hand we would assume `yes_threshold` is >= to pass then we would need an extra flag to define the +1 vote majority semantics and I didn't think it was worth the extra effort and complexity. 
What do you think?

Reply:
My first thought was that a proposal would pass if it's exactly at the threshold, but I'm not an expert on voting standards.  It's up to you in the end!

Reply:
I slept on it and I think you are right, In general case If exact threshold is reached then the vote should pass. 
However for thresholds <= 50% it's not that straightforward because you can have more or the same number of No votes. And in those cases I think the vote should be Defeated. 
In other words if  No > Yes then Proposal is always defeated and  for even splits, let's say 50:50 or 40:40 a tie breaker +1 vote should be required for the vote to Succeed. 
I've made the changes accordingly and it in fact simplified the code which to me is indication that it's the right direction.